sentry/README.rst

```diff
@@ -0,0 +1,177 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+======
+Sentry
+======
+
+This module allows painless `Sentry <https://sentry.io/>`__ integration with
```

Comment:
README should be split according to [OCA's template module](https://github.com/OCA/maintainer-tools/tree/master/template/module/readme)

Reply:
Ok i tried to fix it